src/routes/dieta/destroy.js

```diff
@@ -0,0 +1,10 @@
+module.exports = Dieta => (req, res) => {
+    return Dieta
+    .destroy({
+        where: {
+            id: req.params.dieta_id
```

Comment:
Agora que me dei conta de uma coisa: O que acontece se `req.params.dieta_id` for `undefined`? :thinking:

Reply:
em que situação poderia ser undefined?